src/main/java/edu/kit/ipd/crowdcontrol/objectservice/feedback/FeedbackSender.java

```diff
@@ -0,0 +1,132 @@
+package edu.kit.ipd.crowdcontrol.objectservice.feedback;
+
+import edu.kit.ipd.crowdcontrol.objectservice.database.model.tables.records.AnswerRecord;
+import edu.kit.ipd.crowdcontrol.objectservice.database.model.tables.records.ExperimentRecord;
+import edu.kit.ipd.crowdcontrol.objectservice.database.model.tables.records.RatingRecord;
+import edu.kit.ipd.crowdcontrol.objectservice.database.operations.AnswerRatingOperations;
+import edu.kit.ipd.crowdcontrol.objectservice.database.operations.ExperimentOperations;
+import edu.kit.ipd.crowdcontrol.objectservice.database.operations.WorkerOperations;
+import edu.kit.ipd.crowdcontrol.objectservice.mail.MailHandler;
+import edu.kit.ipd.crowdcontrol.objectservice.mail.MailSender;
+import edu.kit.ipd.crowdcontrol.objectservice.template.Template;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jooq.Result;
+
+import javax.mail.MessagingException;
+import java.io.*;
+import java.util.*;
+
+/**
+ * Sends feedback to workers according to their answers.
+ *
+ * @author Felix Rittler
+ */
+public class FeedbackSender {
+
+    private static final Logger LOGGER = LogManager.getLogger(FeedbackSender.class);
+    private MailSender handler;
+    private AnswerRatingOperations answerOps;
+    private ExperimentOperations expOps;
+    private WorkerOperations workerOps;
+
+    public FeedbackSender(MailHandler handler, AnswerRatingOperations answerOps, ExperimentOperations expOps, WorkerOperations workerOps) {
+        this.handler = handler;
+        this.answerOps = answerOps;
+        this.expOps = expOps;
+        this.workerOps = workerOps;
+    }
+
+    public String getFeedback(int expId, int workerId) throws FeedbackException {
+
+        LOGGER.trace("Started creating feedback message to worker " + workerId + ".");
+
+        String feedbackMessage = loadMessage("src/main/resources/feedback/feedbackMessage.txt");
+        String feedbackAnswer = loadMessage("src/main/resources/feedback/feedbackAnswer.txt");
+        String feedbackRating = loadMessage("src/main/resources/feedback/feedbackRating.txt");
+
+        List<AnswerRecord> answers = answerOps.getAnswersOfExperimentOfWorker(expId, workerId);
```

Comment:
maybe try to get the experimentRecord before, so you dont have the case here that the experimentId can be invalid.